codespell_lib/_codespell.py

```diff
@@ -343,29 +386,41 @@ def parse_options(args):
     parser.add_argument('-x', '--exclude-file', type=str, metavar='FILE',
                         help='ignore whole lines that match those '
                              'in the file FILE. The lines in FILE '
-                             'should match the to-be-excluded lines exactly')
-
+                             'should match the to-be-excluded lines exactly',
+                        ).complete = shtab.FILE
+
+    interactive_opts = {
+        -1: '',
+        0: 'no interactivity.',
+        1: 'ask for confirmation.',
+        2: 'ask user to choose one fix when more than one is available.',
+        3: 'both 1 and 2',
```

Comment:
I wonder if for consistency we should remove 3 and just call it out in the example and list the binary flags stuff again?

Reply:
I'll fix it.